CONTRIBUTING.md

```diff
@@ -1,14 +1,31 @@
 # Contributing
 1. Fork this repository to your own account
-2. Make your changes and verify that tests pass
-3. Commit your work and push to a new branch on your fork
-4. Submit a pull request
-5. Participate in the code review process by responding to feedback
+2. Make your changes and verify that tests pass (with `check` gradle task)
+3. Run `diktat` static analyzer with `diktatFix` gradle task or `diktatCheck` task depending on what you want.
+4. Run `detekt` with `detekt` gradle task
```

Comment:
I think, it's better to use `detektAll`, because AFAIR `detekt` task is only for JVM

Reply:
guess you are right, will fix it